graphql2/graphqlapp/escalationpolicy.go

```diff
@@ -323,6 +324,30 @@ func (ep *EscalationPolicy) Steps(ctx context.Context, raw *escalation.Policy) (
 	return ep.PolicyStore.FindAllSteps(ctx, raw.ID)
 }
 
+func (ep *EscalationPolicy) Notices(ctx context.Context, raw *escalation.Policy) ([]graphql2.Notice, error) {
```

Comment:
Can we map this to `notice.Notice` instead (inside `gqlgen.yml`)? We could probably just do `return ep.NoticeStore.FindAllPolicyNotices(ctx, raw.ID)` that way without needing to loop over them to convert.

The only one we may have trouble with is the `Type` but we can map that with a marshaler method (can add UnmarshalGQL method like in assignment/targetttype.go) if it complains.